src/helpers/investApi.js

```diff
@@ -13,11 +13,36 @@ exports.fetchTagDesc = async (tag) => {
     json: true,
   };
 
-  return await axios
-    .get(url, options)
-    .then((response) => response.data.items[0].excerpt)
+  const response = await axios.get(url, options)
+    .then((json) => json.data.items)
     .catch((err) => {
       console.log('error:', err);
-      return `A ${tag} is a keyword or term assigned to a piece of information`;
-    });
+    })
+  
+  return response;
 };
+
+/**
+ * @description - This function preapares an array of objects to be inserted into tags schema
+ * @param {tags} - array of strings
+ * @param {response} - array of objects
+ * @returns {resp} - array of objects
+ */
+exports.prepareTags = (tags, response) => {
+  const resp = [];
+  for(let i=0; i < tags.length; i++) {
```

Comment:
Use forEach or for of, they are more clean

Reply:
Correct, making a change.